app/lib/rocketchatapi.js

```diff
@@ -0,0 +1,26 @@
+// using env variables for now, should be replaced with cookies!
```

Comment:
put this under the component's own `lib` directory - not the global one

Reply:
done